lib/blocs/feature_flag/feature_flag_bloc.dart

```diff
@@ -22,9 +21,9 @@ class FeatureFlagBloc extends Bloc<FeatureFlagEvent, FeatureFlagState> {
     required this.featureFlagClient,
   }) : super(const FeatureFlagState()) {
     on<_Initialize>(_onInitFlag);
-    on<_Fetch>(_onFetchFlag);
-    on<_Reload>(_onReloadFlag);
+    on<_Load>(_onLoadFlags);
     on<_Register>(_onRegisterFlag);
+    on<_ChangeIdentity>(_changeIdentityFlag);
```

Comment:
Do you use this event?

Reply:
not for now, but maybe later I will add change user method, without complete log out

Reply:
that's why I may need it, but not sure for now